perspective_reddit_bot/README.md

```diff
@@ -0,0 +1,44 @@
+# Perspective Moderation Bot
+
+## Description
+
+This repository contains a python script for a Reddit bot which uses the Perspective API to help moderate comments on a subreddit. By specifying rules in the rules.yaml file, the bot will check new comments on a given subreddit for triggering these rules and can report those that do. In order for the actions to be applied, the reddit credentials provided to the bot must have moderation permissions on the subreddit being moderated.
```

Comment:
nit: wrap to 80 chars, and make rules.yaml into: \`rules.yaml\`

Reply:
tip: For line wrapping, visual studio code and other editors typically have some nice tools to auto-wrap for you.